layers/eight_mile/pytorch/layers.py

```diff
@@ -310,6 +310,15 @@ def __init__(self):
     def forward(self, x):
         return torch.nn.functional.gelu(x)
 
+#Code taken from: https://github.com/huggingface/transformers/blob/766d4bf7920213bdd8a8afb42a72719190124568/src/transformers/activations.py#L27
+class NewGELUActivation(nn.Module):
```

Comment:
What about TF?  dont we need the handling there?

Reply:
The gelu in mead tf layers(https://github.com/dpressel/mead-baseline/blob/bd9cd02c0a1d9c0df91aca171774a6967e6ce190/layers/eight_mile/tf/layers.py#L158) has the same equation as (https://github.com/huggingface/transformers/blob/766d4bf7920213bdd8a8afb42a72719190124568/src/transformers/activations.py#L27), hence i didnt implement it for tf. However, i still have to map new_gelu to gelu in get_activation , as the default is relu